lib/purchase_processor/currency_helper.rb

```diff
@@ -1,12 +1,3 @@
-require 'money/bank/google_currency'
-
-# Set the seconds after than the current rates are automatically expired
-# By default, they never expire
-Money::Bank::GoogleCurrency.ttl_in_seconds = 86400
-
-# Set default bank to instance of GoogleCurrency
-Money.default_bank = Money::Bank::GoogleCurrency.new
-
 module CurrencyHelper
```

Comment:
do we need to require anything for this new gem or does the initializer in dealer take care of that?